src/server/routes/changePassword.ts

```diff
@@ -68,6 +69,9 @@ router.get(
   `${Routes.CHANGE_PASSWORD}${Routes.CHANGE_PASSWORD_TOKEN}`,
   async (req: Request, res: ResponseWithRequestState) => {
     const { token } = req.params;
+    res.locals.pageData.browserName = getBrowserNameFromUserAgent(
```

Comment:
We should avoid mutation. This will become forbidden by linting soon.

Reply:
there's a mutation just below this that will be affected too for the immutability changes. This will conflict with your work if you've started changing these files.

```
res.locals.pageData.email = await validateToken(token, req.ip);
```